### PR TITLE
Update the gradle file for sequentially execution

### PR DESCRIPTION
**Pull request by bam-hbt**
fix https://github.com/jenkinsci/peass-ci-plugin/issues/187
Updating exceution mode system propeties in the gradle file for sequentially execution.

### CODE REVIEW

File: dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java [hunk 1]
```diff
@@ -89,9 +87,16 @@ public void visitMethodCallExpression(final MethodCallExpression call) {
...
                ArgumentListExpression arguments = (ArgumentListExpression) call.getArguments();
-               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments);
+               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments, testExecutionProperties);
                testSystemPropertiesLine = propertiesLine;
             }
+         } else if (call.getMethodAsString().equals("integrationTest")) {
+            testLine = call.getLastLineNumber() + offset;
+            if (call.getArguments() instanceof ArgumentListExpression) {
+               ArgumentListExpression arguments = (ArgumentListExpression) call.getArguments();
+               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments, integrationtestExecutionProperties);
```
Comment (DaGeRe) [inline dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java:97]: Why don't you assign ` integrationTestSystemPropertiesLine`  directly?

File: dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java [hunk 2]
```diff
@@ -121,35 +126,58 @@ public void visitMethodCallExpression(final MethodCallExpression call) {
...
+               if (statement instanceof ReturnStatement) {
+                  Expression expression = ((ReturnStatement) statement).getExpression();
+                  if (expression.getText().contains("systemProperties") || expression.getText().contains("systemProperty")) {
+                     potentialSystemProperties = new ExpressionStatement(((ReturnStatement) statement).getExpression());
+                  }
+               } else if (statement instanceof ExpressionStatement) {
+                  potentialSystemProperties = (ExpressionStatement) statement;
+               }
+               if (potentialSystemProperties != null && potentialSystemProperties.getExpression() instanceof MethodCallExpression) {
+                  MethodCallExpression methodCallExpression = (MethodCallExpression) potentialSystemProperties.getExpression();
```
Comment (DaGeRe) [inline dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java:146]: This is (way) to big and should be extracted to a separate method. The rule of thumb is 10 lines are too much (which is not always considered, but should be in new code).

**Comment by DaGeRe** [comment]
Thanks for the PR. Besides the things I've mentioned, this looks fine to me.

File: dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java (hunk 2 above)
Comment (bam-hbt) [reply dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java]: This is refactored

File: dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java (hunk 1 above)
Comment (bam-hbt) [reply dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java]: This is refactored

**Comment by DaGeRe** [comment]
I am not completely sure whether this will work correctly, since the old version did not contain any `systemProperty` statement. But since you are the gradle expert, I'll just merge it and if this works for your CI processes, it's fine.

**Pull request merged**